src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml.cs

```diff
@@ -115,6 +118,11 @@ public async Task<IActionResult> OnGetAsync(string id, string revisionId = null)
 
         private CodeLineModel[] CreateLines(CodeDiagnostic[] diagnostics, InlineDiffLine<CodeLine>[] lines, ReviewCommentsModel comments)
         {
+            if (ShowDiffOnly)
+            {
+                lines = lines.Where(l => l.Kind != DiffLineKind.Unchanged).ToArray();
```

Comment:
I think showing the diff without any context doesn't make it very useful. Look at how github does it there are a few lines of context before/after the section of the diff:

![image](https://user-images.githubusercontent.com/1697911/104368254-adc17680-54d0-11eb-9661-6151b499d456.png)

Reply:
Let me check if I can add few before and after lines to show the context also. This option will anyway be able to narrow down and find the diff. To get actual navigation reference and etc to work user will have to see full review.

Reply:
I have made few changes to show context before and after the diff though not yet pushed to this PR. But thought of getting your feedback. 

![image](https://user-images.githubusercontent.com/55455725/104531351-d3c74380-55c2-11eb-8736-c044b03a8388.png)

Reply:
Very nice! love it.